app/Console/Kernel.php

```diff
@@ -25,6 +25,7 @@ class Kernel extends ConsoleKernel
         Commands\ReviewSignup::class,
         Commands\SendToQuasar::class,
         Commands\ForceDeleteTestRecords::class,
+        Commands\GDPRComplianceCommand::class,
```

Comment:
Totally unrelated to this pull request, but we can have Laravel auto-load these (see [Northstar](https://github.com/DoSomething/northstar/blob/742e3de1d46621926b540cd2052456e08d500e70/app/Console/Kernel.php#L20-L29)'s kernel).

Reply:
Yes, I love this!!